### ISSUE TITLE
Add Undo/Redo

### ISSUE THREAD

**Issue by harbiad** [frappe/print_designer#190]
If you delete something you can't undo that

**Comment by moizsami** [comment]
Yes would appreciate this feature, as the application is still a bit buggy sometimes. when I am trying to shift a cell suddenly few areas get erased, it would be good if we had an undo option.

**Comment by the3dfox** [comment]
+1 highly needed

**Comment by naga-mantha** [comment]
+1

**Comment by jslocomotor** [comment]
+1 , please
